app/javascript/mastodon/components/status_action_bar.js

```diff
@@ -301,13 +302,14 @@ class StatusActionBar extends ImmutablePureComponent {
 
       if (account.get('acct') !== account.get('username')) {
         const domain = account.get('acct').split('@')[1];
+        const prettyDomain = domain ? punycode.toUnicode(domain) : domain;
```

Comment:
`account.get('acct')` already returns the “prettified” acct, so this isn't needed here